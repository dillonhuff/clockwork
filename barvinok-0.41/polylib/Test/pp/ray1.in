1 4
1 1 -1 0 # x >= a

1 3
1 2 -11

a
