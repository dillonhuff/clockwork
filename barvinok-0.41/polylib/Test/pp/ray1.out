---------------------------------------
Overall rays :
Ray:    [   1  ]
---------------------------------------
Domain :
         2a  -11 >= 0
          1 >= 0

Vertices :
[ a ]
